test/load/terraform/main.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
+  required_version = ">= 0.13"
```

Comment:
Can you put an AWS provider version constraint in this as well? That would help in case the AWS provider was updated to something that was incompatible with the TF code, we could slowly make modifications until we are ready to move up versions rather than CI pulling latest all the time.